keyboard/android/src/main/java/com/capacitorjs/plugins/keyboard/Keyboard.java

```diff
@@ -60,7 +61,11 @@ public Keyboard(AppCompatActivity activity, boolean resizeOnFullScreen) {
         //calculate density-independent pixels (dp)
         //http://developer.android.com/guide/practices/screens_support.html
         DisplayMetrics dm = new DisplayMetrics();
-        activity.getWindowManager().getDefaultDisplay().getMetrics(dm);
+        if (android.os.Build.VERSION.SDK_INT >= 30) {
```

Comment:
```suggestion
        if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
```

use the constant instead of the hardcoded value